packages/salesforcedx-vscode-core/src/commands/baseDeployRetrieve.ts

```diff
@@ -195,14 +295,22 @@ export abstract class DeployExecutor<T> extends DeployRetrieveExecutor<T> {
 }
 
 export abstract class RetrieveExecutor<T> extends DeployRetrieveExecutor<T> {
+  public readonly sourceConflictErrorInfoMsg =
```

Comment:
Can this be moved to ../messages/i18n.ts?

Reply:
I revisited this and decided to remove it.  There is no equivalent console.info msg for the retrieve operation when retrieving against a non-src tracked org so it makes the operations slightly different depending on org.  Plus it seems kinda confusing to get an info msg about the conflicts but then still overwrite the file.  Happy to discuss further.